.env

```diff
@@ -0,0 +1,5 @@
+COMPOSE_PROJECT_NAME=journal
+COMPOSER_DEV_ARG=
+IMAGE_TAG=develop
+APP_ENV=dev
+PHP_VERSION=2f00bfb3e9385f38997f04cea41ffbd3ff9d10a8
```

Comment:
not sure if it's possible to add a comment in this file, but a link or some tracking of what tag this was pinned from could be useful

Reply:
I guess it's our base image so the only interesting thing about that could be the date of generation